config/site.php

```diff
@@ -140,11 +140,11 @@
                 ],
                 'documentation' => [
                     'book' => [
-                        'url' => 'http://book.cakephp.org/',
+                        'url' => 'http://book.cakephp.org/4',
```

Comment:
There's no need to change this. The manual homepage will always be for current major release. Just the protocol should be changed to https.

Reply:
Done